custom_functions/container_merge.py

```diff
@@ -31,17 +34,26 @@ def container_merge(target_container=None, container_list=None, workbook=None, c
     
     container_url = phantom.build_phantom_rest_url('container', container['id'])
     
+    
     # Check if container_list input is a list of IDs
-    if isinstance(container_list, list) and (all(isinstance(x, int) for x in container_list) or all(x.isnumeric() for x in container_list)):
-        pass
+    if isinstance(container_list, str):
+        container_list = container_list.replace('[','').replace(']','').split(',')
```

Comment:
I think we should add `replace(' ','')` like we do in other custom functions. Without it the placeholder text is not valid, because lists with spaces like the one below will not get parsed:

`[1, 5, 10]`

Reply:
Updated this line of code in latest commit